mayastor/src/bdev/nexus/nexus_bdev.rs

```diff
@@ -874,6 +875,38 @@ impl Nexus {
         }
     }
 
+    pub(crate) fn flush(
+        &self,
+        pio: *mut spdk_bdev_io,
+        channels: &NexusChannelInner,
+    ) {
+        let mut io = Bio(pio);
+        io.ctx_as_mut_ref().in_flight = channels.ch.len() as i8;
+        let results = channels
+            .ch
+            .iter()
+            .map(|c| unsafe {
+                let (b, c) = c.io_tuple();
```

Comment:
Reusing 'c' is a tad confusing, (bdev, chan) as used in reset is more readable.

Reply:
Fixed https://github.com/openebs/Mayastor/pull/291/commits/4b75f677cb11fb839585765031c99f207e4bd770